### PR TITLE
*: Use tidb/kv.ErrLockConflict instead of fmt.Errorf for lock conflict

### PR DESCRIPTION
**Pull request by shenli**

### CODE REVIEW

File: transaction.go [hunk 1]
```diff
@@ -461,7 +462,7 @@ func (txn *Txn) batchPrewriteSecondaryRowsWithLockClean(tbl []byte, rowMs map[st
 				return errors.Trace(err)
 			}
 			if lock != nil {
-				return fmt.Errorf("can't clean lock, column:%+v; conflict lock: %+v, lock ts: %d", lock.getColumn(), lock, lock.getTimestamp())
```
Comment (siddontang) [inline transaction.go]: add a warn or error log here?

**Comment by c4pt0r** [comment]
LGTM

File: transaction.go (hunk 1 above)
Comment (shenli) [inline transaction.go]: ok

Comment (siddontang) [inline transaction.go]: can't use %v for lock.GetColumn, it returns binary data,and console will make a "peng" noise .

**Comment by siddontang** [comment]
LGTM

**Pull request merged**